processing/src/main/java/org/apache/druid/query/QueryMetrics.java

```diff
@@ -262,36 +265,58 @@
 
   /**
    * Registers "query time" metric.
+   *
+   * Measures the time between a server thread starting to handle a query, and the response being fully written to
+   * the response output stream. Does not include time spent waiting in the server queue to acquire a thread.
    */
   QueryMetrics<QueryType> reportQueryTime(long timeNs);
 
   /**
    * Registers "query bytes" metric.
+   *
+   * Measures the total number of bytes written by the query server thread to the response output stream.
+   *
+   * Emitted once per query.
    */
   QueryMetrics<QueryType> reportQueryBytes(long byteCount);
 
   /**
-   * Registeres "segments queried count" metric.
+   * Registers "segments queried count" metric.
    */
   QueryMetrics<QueryType> reportQueriedSegmentCount(long segmentCount);
 
   /**
    * Registers "wait time" metric.
+   *
+   * Measures the total time segment-processing runnables spend waiting for execution in the processing thread pool.
+   *
+   * Emitted once per segment.
    */
   QueryMetrics<QueryType> reportWaitTime(long timeNs);
 
   /**
    * Registers "segment time" metric.
+   *
+   * Measures the total wall-clock time spent operating on segments in processing threads.
+   *
+   * Emitted once per segment.
    */
   QueryMetrics<QueryType> reportSegmentTime(long timeNs);
 
   /**
    * Registers "segmentAndCache time" metric.
+   *
+   * Measures the total wall-clock time spent in processing threads, either operating on segments or retrieving items
+   * from cache.
+   *
+   * Emitted once per segment.
    */
   QueryMetrics<QueryType> reportSegmentAndCacheTime(long timeNs);
 
   /**
    * Registers "cpu time" metric.
+   *
+   * Measures the total amount of CPU time spent by all threads (server and processing) involved in a query.
```

Comment:
Will it be clearer if you mention that this metric includes segment processing wait time, segment processing time, and per-segment result merge time? I guess `server` seems a bit ambiguous to me since there are so many severs in druid.